specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-08-01/netapp.json

```diff
@@ -0,0 +1,2415 @@
+          "maxLength": 36,
+          "minLength": 36,
+          "example": "9560acf5-4e3a-12e7-9bdb-02007cca7779"
+        },
+        "subnetId": {
+          "type": "string",
+          "description": "The Azure Resource URI for a delegated subnet. Must have the delegation Microsoft.NetApp/volumes"
+        },
+        "mountTargets": {
+          "title": "mountTargets",
+          "description": "List of mount targets",
+          "items": {
+            "$ref": "#/definitions/mountTargetList"
+          }
+        },
+        "volumeType": {
+          "type": "string",
+          "description": "What type of volume is this",
+          "example": "DataProtection"
+        },
+        "dataProtection": {
+          "title": "DataProtection",
+          "description": "DataProtection volume, can have a replication object",
+          "properties": {
+            "replication": {
+              "title": "Replication",
+              "description": "Replication properties",
+              "type": "object",
+              "$ref": "#/definitions/replicationObject"
+            }
+          }
+        }
+      }
+    },
+    "exportPolicyRule": {
+      "description": "Volume Export Policy Rule",
+      "type": "object",
+      "properties": {
+        "ruleIndex": {
+          "type": "integer",
+          "description": "Order index"
+        },
+        "unixReadOnly": {
+          "type": "boolean",
+          "description": "Read only access"
+        },
+        "unixReadWrite": {
+          "type": "boolean",
+          "description": "Read and write access"
+        },
+        "cifs": {
+          "type": "boolean",
+          "description": "Allows CIFS protocol"
+        },
+        "nfsv3": {
+          "type": "boolean",
+          "description": "Allows NFSv3 protocol"
+        },
+        "nfsv41": {
+          "type": "boolean",
+          "description": "Allows NFSv4.1 protocol"
+        },
+        "allowedClients": {
+          "type": "string",
+          "description": "Client ingress specification as comma separated string with IPv4 CIDRs, IPv4 host addresses and host names"
+        }
+      }
+    },
+    "replicationObject": {
+      "description": "Replication properties",
+      "type": "object",
+      "required": [
+        "endpointType",
+        "remoteVolumeResourceId",
+        "replicationSchedule"
+      ],
+      "properties": {
+        "replicationId": {
+          "title": "replicationId",
+          "type": "string",
+          "description": "Id"
+        },
+        "endpointType": {
+          "title": "endpointType",
+          "type": "string",
+          "description": "Indicates whether the local volume is the source or destination for the Volume Replication",
+          "example": "src, dst"
+        },
+        "replicationSchedule": {
+          "title": "replicationSchedule",
+          "type": "string",
```

Comment:
>string [](start = 19, length = 6)

Should this be an enum? (Using the x-ms-enum extension you can still model the enum as a string, but the generated SDKs will include a class with constants representing each possible value. It makes it more discoverable for the SDK user.)

Reply:
Yeah, in JSON enums look like strings. I'm ok leaving it for later.